include/tlog/tap.h

```diff
@@ -32,18 +32,24 @@
 #include <termios.h>
 #include <unistd.h>
 #include <time.h>
+#include <utmpx.h>
+
+#define TAP_LINESIZE (sizeof(((struct utmpx *)0)->ut_line))
+#define TAP_DEVPATH "/dev/"
+#define TAP_DEVSIZE (sizeof(TAP_DEVPATH) - 1)
```

Comment:
To follow convention, can we use names prefixed with TLOG_ such as:

~~~
   TLOG_TAP_UTMP_LINESIZE
   TLOG_TAP_DEVPATH
   TLOG_TAP_DEVSIZE
~~~